src/actions/profileActions/profileActionCreator.js

```diff
@@ -0,0 +1,33 @@
+import ACTION_TYPE from '../authActions/actionTypes';
+
+export const fetchProfileSuccess = (userData) => ({
+	type: ACTION_TYPE.FETCH_PROFILE_SUCCESS,
+	payload: {
+		username: userData.profile.username,
+		firstName: userData.profile.first_name,
+		lastName: userData.profile.last_name,
+		bio: userData.profile.bio,
+		avatar: userData.profile.avatar
+	}
+});
+
+export const fetchProfileFailure = (error) => ({
+	type: ACTION_TYPE.FETCH_PROFILE_FAILURE,
+	payload: {
+		error
+	}
+});
+
+export const updateProfileSuccess = (message) => ({
+	type: ACTION_TYPE.UPDATE_PROFILE_SUCCESS,
+	payload: {
+		successMessage: message
```

Comment:
maintain the same payload and that is `message: message`. 
This is regardless of whether we have an error or whether the operation is successful

Reply:
Resolved